modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -2265,8 +2266,106 @@ var UIDynamicList = UIElement.extend(/** @lends LuCI.ui.DynamicList.prototype */
 			this.addItem(dl, this.values[i], label);
 		}
 
+		this.initDragAndDrop(dl);
+
 		return this.bind(dl);
 	},
+	initDragAndDrop: function(dl) {
+		let draggedItem = null;
+		let touchStartY = 0;
+		let placeholder = null;
+
```

Comment:
add this for later
```js

		const resetStyles = (target) => {
			if (target?.classList.contains('item')) {
				target.style.opacity = "";
				target.style.borderTop = "";
			}
		};
```

Reply:
I did another solution, I added CSS styles, and dynamically added a "dragging" class and "drag-over" class.  Is even more elegant.